src/Text/Pandoc/Lua/Marshal/Reference.hs

```diff
@@ -0,0 +1,101 @@
+{-# LANGUAGE LambdaCase           #-}
```

Comment:
There's a bunch of stuff in this commit, like this for example, that doesn't relate to the change.
Can you rebase on current master HEAD?

Reply:
I'm sorry, we forgot to do that, it's now rebased